security/q-feeds-connector/src/opnsense/scripts/unbound/blocklists/qfeeds_bl.py

```diff
@@ -53,3 +57,14 @@ def get_blocklist(self):
 
     def get_passlist_patterns(self):
         return []
+    
+    def _is_enabled(self):
```

Comment:
I don't mind adding this for now, but eventually it might be better to just check the filesize of `/usr/local/etc/unbound/qfeeds-blocklists.conf` to reduce overhead. qfeedctl is doing the same now